packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagelist/_mixins.scss

```diff
@@ -1,11 +1,19 @@
 @use "sass:math";
 
-@mixin chat-messagelist($padding) {
+@mixin chat-messagelist(
+  $padding,
+  $date-header-color,
+) {
   .dx-chat-messagelist {
     .dx-scrollable-content {
       padding-inline: $padding;
     }
   }
+
+  .dx-chat-messagelist-date-header {
+    padding: $padding;
```

Comment:
Please use different paddings here:

padding-top: 20px for the first header, and padding-bottom: 12px for all header elements.

Reply:
updated, also added variables for these paddings